runtime/polkadot/src/lib.rs

```diff
@@ -1857,7 +1857,9 @@ sp_api::impl_runtime_apis! {
 			parachains_runtime_api_impl::validation_code_hash::<Runtime>(para_id, assumption)
 		}
 
-		fn disputes() -> Vec<(u32, CandidateHash, DisputeState)> { todo!() }
+		fn disputes() -> Vec<(u32, CandidateHash, DisputeState)> {
+			runtime_parachains::runtime_api_impl::vstaging::get_session_disputes::<Runtime>()
```

Comment:
Which brings me to a second thing, have we tested this on Versi already? If not, @tdimitrov could you help @s0me0ne-unkn0wn with that please?

Reply:
We haven't yet, I was planning that for tomorrow afternoon, and it would be really great to have someone who can help with that.

Btw, should we bring `ExecutorParams` to `v3` also? Right now it is in `vstaging` primitives, but it is encapsulated into `SessionInfo`, which is a stable `v3` primitive.

Reply:
Yes. We test on Versi, if everything works as expected, we go right to production.